packages/components/src/disabled/index.native.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+function Disabled( { children } ) {
+	return <View pointerEvents="none">{ children }</View>;
+}
```

Comment:
This component doesn't match the same functionality as the web version but does the minimum expected that is preventing the user interaction, what do you think?

If you rather have this completely implemented I could move this logic to the Inserter menu component where I'm using it.

Reply:
I think this looks fine 👍